tests/load/pipeline/test_redshift.py

```diff
@@ -41,3 +45,71 @@ def my_source() -> Any:
         assert pip_ex.value.__cause__.data_type == "time"
     else:
         assert pip_ex.value.__cause__.data_type in ("time", "binary")
+
+
+AWS_BUCKET = dlt.config.get("tests.bucket_url_s3", str)
+redshift_with_staging_configs = [
+    DestinationTestConfiguration(
+        destination_type="redshift",
+        staging="filesystem",
+        file_format="jsonl",
+        bucket_url=AWS_BUCKET,
+        extra_info="credential-forwarding",
+    ),
+    DestinationTestConfiguration(
+        destination_type="redshift",
+        staging="filesystem",
+        file_format="jsonl",
+        bucket_url=AWS_BUCKET,
+        staging_iam_role="arn:aws:iam::267388281016:role/redshift_s3_read",
+        extra_info="s3-role",
+    ),
+    ## for parquet, region should be unset
+    DestinationTestConfiguration(
+        destination_type="redshift",
+        staging="filesystem",
+        file_format="parquet",
+        bucket_url=AWS_BUCKET,
+        staging_iam_role="arn:aws:iam::267388281016:role/redshift_s3_read",
+        extra_info="s3-role",
+    ),
+]
+staging_regions = ["eu-central-1", ""]
+test_cases = list(itertools.product(redshift_with_staging_configs, staging_regions))
+
+
+@pytest.mark.parametrize("destination_config, staging_region", test_cases)
+def test_copy_from_staging_with_region(
```

Comment:
this test is not really needed. we have plenty of tests like this and they will add `REGION` clause to the copy command. if you want to write a proper test:
1. we'd need to setup cross - region bucket (too much work)
2. you can write a unit test but that needs our code to be restructured. See how Snowflake destination is doing that (we separate sql generating part so it can be unit tested). ping me if you want to do it. but first make standard test to pass